examples/stm32/f4/nucleo-f401re/nucleo-f401re.ld

```diff
@@ -0,0 +1,32 @@
+/*
+ * This file is part of the libopencm3 project.
+ *
+ * Copyright (C) 2009 Uwe Hermann <uwe@hermann-uwe.de>
+ * Copyright (C) 2011 Stephen Caudle <scaudle@doceme.com>
```

Comment:
There's nothing here written by either of these people of any relevance.  Either omit it, or put your own name here for 2015/2016